nsis/src/nsis/format/NsisCommonHeader.java

```diff
@@ -12,6 +12,33 @@
 public class NsisCommonHeader implements StructConverter {
 	private int flags;
 	private NsisBlockHeader[] blockHeaders;
+	private int install_reg_rootkey;
```

Comment:
Since it was done in other files (NsisBlockHeader for example), what do you think about providing good descriptive names to fields here (lb_bg is anything but descriptive) that would also fit the style guide? 
https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names

Reply:
Good idea, it will also stay consistent. In the case that I have no clue what the variable does, should I leave the name and just make sure to follow the style guide? For example 'installRegRootkey',  I don't know of a better name because I don't know what the variable does

Reply:
Sounds good enough.